docs/submission/clinical-data-validation-rules.md

```diff
@@ -19,29 +19,28 @@ The data dictionary contains certain data elements regarded as "identifiers". Th
 - Treatment: `submitter_treatment_id`
 - Follow Up: `submitter_follow_up_id`
 
-These fields must to be coded specifically for ICGC-ARGO purposes using the following rules:
+These fields must to be coded specifically for ICGC ARGO purposes using the following rules:
 
-- These identifiers should not be derived from IDs such as biobank or hospital identifiers or any other personal identifying information. These IDs are to be coded in such a way that they cannot be tracked back to the individual donors, except by the submitting program. Only the program will keep the key that will permit the data to be linked back to the individual donors. This key must not be communicated to the data users.
+- These identifiers should not be derived from biobank or hospital identifiers or any other personal identifying information. These IDs are to be coded in such a way that they cannot be tracked back to the individual donors, except by the submitting program. Only the program will keep the key that will permit the data to be linked back to the individual donors. This key must not be communicated to the data users.
 - Identifiers are assigned by each submitting program and must be unique within all the data submitted by that program (no duplicate IDs allowed).
-- These identifiers should be values that uniquely identify the entity they refer to.
-- Identifiers referring to the same entity should be consistent across different submissions from the submitting program, and should not be re-used to refer to two different entities. For example, the same donor should not be assigned different identifiers in different files or subsequent data submissions.
-- Identifiers cannot begin with DO, SP, SA, PD, TR or FU. These prefixes are reserved for ICGC-ARGO IDs.
+- Identifiers referring to the same entity should be consistent across separate program submissions and should not be re-used for different entities. For example, the same donor should not be assigned different identifiers in different files or subsequent data submissions.
+- Identifiers cannot begin with DO, SP, SA, PD, TR or FU. These prefixes are reserved for ICGC ARGO IDs.
```

Comment:
for now (until we change anything, should we make a note about `case-sensitivity`?

Reply:
Good point @rosibaj - so right now, the IDs are still case-sensitive, right? I'll add a note about this.

Reply:
yes right now they still are  - so at least a notice/warning for now until we change that feature.